components/layout_2020/display_list/stacking_context.rs

```diff
@@ -192,25 +193,70 @@ impl DisplayList {
     }
 }
 
-pub(crate) struct StackingContextFragment {
-    scroll_node_id: ScrollTreeNodeId,
-    clip_chain_id: wr::ClipChainId,
-    section: StackingContextSection,
-    containing_block: PhysicalRect<Length>,
-    fragment: ArcRefCell<Fragment>,
+pub(crate) enum StackingContextFragment {
```

Comment:
Maybe this should be renamed to something like `StackingContextMember`. I think it would be good to put a little documentation here about why InlineStackingContexts need to be stored in the same data structure. I assume they need to maintain their order.

Reply:
done in 8e60b20ca11c2a3d755e21931661c14d108f9261